frontends/web/src/routes/account/send/send.tsx

```diff
@@ -91,7 +91,7 @@ interface State {
     amountError?: string;
     dataError?: string;
     paired?: boolean;
-    noMobileChannelError: boolean;
+    noMobileChannelError?: boolean;
```

Comment:
Why is it necessary to make it optional? Easier to keep like before, but fix the assignment to be `false` instead of `undefined`.

reason: to states for the var are sufficient, don't need three states where two of them mean the same.

Reply:
Feel free to do so. If I remember correctly, the `undefined` came from the `account` in https://github.com/digitalbitbox/bitbox-wallet-app/blob/16d6924663ce9088f55677aa8ab848c6f468925f/frontends/web/src/routes/account/send/send.tsx#L146

And for consistency, you could/should then also change this for `paired` and other fields. 🤓

Reply:
:man_shrugging: it's your PR.